pkg/metrics/metrics.jsx

```diff
@@ -67,6 +67,9 @@ const _ = cockpit.gettext;
 // format Date as YYYY-MM-DD HH:mm:ss UTC which is human friendly and systemd compatible
 const formatUTC_ISO = t => `${t.getUTCFullYear()}-${t.getUTCMonth() + 1}-${t.getUTCDate()} ${t.getUTCHours()}:${t.getUTCMinutes()}:${t.getUTCSeconds()} UTC`;
 
+// podman's system containers cgroup
+const podmanSystemRe = /libpod-(?<containerid>[a-z|0-9]{64})\.scope$/;
```

Comment:
It's also being used for user containers, just inside `user.slice` instead of `system.slice`. So the "System" bit is a bit misleading.

Reply:
Excellent point, let's rename it